src/components/TableCell/TableCell.tsx

```diff
@@ -1,6 +1,14 @@
 import classNames from "classnames";
 import PropTypes from "prop-types";
-import React from "react";
+import React, { HTMLProps, ReactNode } from "react";
+
+type Props = {
+  children?: ReactNode;
+  className?: string;
+  expanding?: boolean;
+  hidden?: boolean;
+  role?: string;
```

Comment:
Is there an enum for roles in this context?

Also, see comment above about documenting default value for optional prop.

Reply:
I don't think there is in any context, react defines `role` in `HTMLAttributes` as `role?: string`.